FUNDING.yml

```diff
@@ -1 +1,2 @@
-github: [happihacking]
```

Comment:
O_o

Reply:
Apparently, even if this is rejected, this is how the contributor profile looks like:

![Screenshot 2020-10-08 at 09 43 44](https://user-images.githubusercontent.com/91769/95429585-e6d6d180-094a-11eb-9106-4786ba293cb9.png)

But at least it now points to happihacking.

Reply:
It looks like the book is forked for a Chinese translation, which is a nice thing:

https://github.com/hitdavid/theBeamBook

So, let's just assume this was unintentional :)

Reply:
OMG
I didn't want to commit the code this way on purpose.
It’s a BIG mistake.
really sorry to author and contributors.

I want to get some infomation about potential Chinese-Version book readers through My sponsorship button, and totally do not intent to change happi’s funding config.
Sorry again.